monitoring/ceph-mixin/alerts.libsonnet

```diff
@@ -0,0 +1,875 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'cluster health',
+        rules: [
+          {
+            alert: 'CephHealthError',
+            expr: 'ceph_health_status == 2',
```

Comment:
Given that we're not using any jsonnet preprocessor magic here and that YAML is far more readable/user-friendly, why not passing through the original YAML, or if really needed, generating this with the [`std.parseYaml`](https://jsonnet.org/ref/stdlib.html#:~:text=yields%20%7B%20%22foo%22%3A%20%22bar%22%20%7D.-,std.parseYaml(str),-Available%20since%20version)?

I'd go for a jsonnet file here if we had something like Grafonnet, but tbh I find most of the info here cannot be 'compressed' much further:
```py
PrometheusAlerts().addGroup(
    Alert(
      'CephHealthError',
      'ceph_health_status == 2',
      ...
    )
```

Reply:
Ah I don't remember precisely, I probably missed the fact that `std.parseYaml` existed in the first hand and probably saw a bunch of projects doing a similar thing than what I attempted here.

I will try to use `std.parseYaml` and see how it goes, this seems at first glance a good thing to do!

Reply:
Thanks! My first suggestion was to simply bypass the yaml file, as I don't think we were doing any preprocessing in there.

Reply:
Ok so now I remember why I didn't do that. parseYaml was not in any jsonnet release for the past year or so but this has changed because v0.18.0 was released 15 days ago. So fortunately for us we will be able to do that :D!